From 408777992b20c152c5f25a6bfaea497f56bc27dc Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Tue, 6 Aug 2024 15:00:29 +0000
Subject: [PATCH] Set default window size instead of size request

Setting a size request causes that the main window cannot be resized
below that value, which causes that the window does not fit into smaller
screens. Use set_default_size() instead to allow to shrink the window if
needed.
---
 src/ocrfeeder/studio/widgetPresenter.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/ocrfeeder/studio/widgetPresenter.py b/src/ocrfeeder/studio/widgetPresenter.py
index 47a8ca9..b0536aa 100644
--- a/src/ocrfeeder/studio/widgetPresenter.py
+++ b/src/ocrfeeder/studio/widgetPresenter.py
@@ -42,7 +42,7 @@ class MainWindow(Gtk.Window):
 
     def __init__(self):
         Gtk.Window.__init__(self, Gtk.WindowType.TOPLEVEL)
-        self.set_size_request(1280, 720)
+        self.set_default_size(1280, 720)
         self.set_default_icon_name(APP_ID)
         self.main_box = Gtk.VBox()
         self.main_box.show()
-- 
GitLab

